modules/tensor_mechanics/doc/content/source/interfacekernels/CZMInterfaceKernel.md

```diff
@@ -0,0 +1,108 @@
+# CZM InterfaceKernel
+!syntax description /InterfaceKernels/CZMInterfaceKernel
+
+## Description
+
+This class implements traction equilibrium across an interface. Specifically this is a small deformation implementation (geometrical quantities, such as surface normal, are always referred to the initial configuration). A `CZMInterfaceKernel`  acts only on one displacement component and therefore the user must setup N kernel one for each dimension of the problem.
+The `CZMInterfaceKernel` use the traction and traction's spatial derivatives provided by a `CZMMaterial` to compute the appropriate residual and Jacobian.
+The responsibility of the `CZMInterfaceKernel` is to integrate the traction over the interface to compute the Nodal Force and to assemble the proper Jacobian.
+I
```

Comment:
I would not call it Nodal Force because we only compute traction at quadrature points.

Reply:
got it